fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -4,55 +4,52 @@ import com.gu.contentapi.client.model.{Tag, Content}
 import com.gu.facia.api.utils._
 import com.gu.facia.client.models.{SupportingItem, MetaDataCommonFields, Trail, TrailMetaData}
 
-case class ImageReplace(imageSrc: String, imageSrcWidth: String, imageSrcHeight: String)
+case class FaciaImage(
+  imageType: ImageType,
+  imageSrc: String,
+  imageSrcWidth: Option[String],
+  imageSrcHeight: Option[String]
+)
 
-object ImageReplace {
-  def fromTrailMeta(trailMeta: MetaDataCommonFields): Option[ImageReplace] =
-    for {
-      imageReplace <- trailMeta.imageReplace.filter(identity)
-      imageSrc <- trailMeta.imageSrc
-      imageSrcWidth <- trailMeta.imageSrcWidth
-      imageSrcHeight <- trailMeta.imageSrcHeight
-    } yield ImageReplace(imageSrc, imageSrcWidth, imageSrcHeight)
-}
+sealed trait ImageType
+case object Cutout extends ImageType { override def toString = "cutout" }
+case object Replace extends ImageType { override def toString = "replace" }
+case object ImageDefault extends ImageType { override def toString = "default" }
```

Comment:
This makes no sense. What is `ImageDefault`? How can you have a `FaciaImage` with an `ImageDefault`? It will never be `ImageDefault`, because if it ends up (meant to) be(ing) `ImageDefault`, you will just get a `None` for `FaciaImage`.

I am removing this.